kitty/css/style.css

```diff
@@ -0,0 +1,530 @@
+  }
+  .icon-mileage:before {
+  content: "\e626";
+  }
+  .icon-other:before {
+  content: "\e627";
+  }
+  .icon-print:before {
+  content: "\e628";
+  }
+  .icon-record:before {
+  content: "\e629";
+  }
+  .icon-report:before {
+  content: "\e62a";
+  }
+  .icon-reset:before {
+  content: "\e62b";
+  }
+  .icon-save:before {
+  content: "\e62c";
+  }
+  .icon-schedule:before {
+  content: "\e62d";
+  }
+  .icon-chevron-thick-left:before {
+  content: "\e800";
+  }
+  .icon-chevron-thick-right:before {
+  content: "\e801";
+  }
+  .icon-chevron-thick-down:before {
+  content: "\e802";
+  }
+  .icon-chevron-thick-up:before {
+  content: "\e803";
+  }
+  .icon-order:before {
+  content: "\e804";
+  }
+  .icon-edit-log:before {
+  content: "\e805";
+  }
+  .icon-get-started:before {
+  content: "\e806";
+  }
+  .icon-procurify:before {
+  content: "\e807";
+  }
+  .icon-empty-cart:before {
+  content: "\e900";
+  }
+  .icon-attention:before {
+  content: "\e901";
+  }
+  .icon-ap:before {
+  content: "\e902";
+  }
+  .icon-catalog:before {
+  content: "\e903";
+  }
+  .icon-map:before {
+  content: "\e904";
+  }
+  .icon-camera:before {
+  content: "\e905";
+  }
+  .icon-creditcard:before {
+  content: "\e906";
+  }
+  .icon-auditlog:before {
+  content: "\e907";
+  }
+  .icon-checkmark:before {
+  content: "\e908";
+  }
+  .icon-gallery:before {
+  content: "\e909";
+  }
+  .icon-notification:before {
+  content: "\e910";
+  }
+  .icon-sort-ascending:before {
+  content: "\e911";
+  }
+  .icon-sort-descending:before {
+  content: "\e912";
+  }
+  .icon-switch:before {
+  content: "\e913";
+  }
+  .icon-upload:before {
+  content: "\e943";
+  }
+  .icon-chevron-thick-both:before {
+  content: "\e915";
+  }
+  .icon-emptystate:before {
+  content: "\e90a";
+  }
+  .icon-unlock:before {
+  content: "\e90b";
+  }
+  .icon-vacation:before {
+  content: "\e90c";
+  }
+  .icon-audit-arrow:before {
+  content: "\e90d";
+  }
+  .icon-timer:before {
+  content: "\e90e";
+  }
+  .icon-disable:before {
+  content: "\e90f";
+  }
+  .icon-pointer-down:before {
+  content: "\e914";
+  }
+  .icon-pointer-up:before {
+  content: "\e916";
+  }
+  .icon-pointer-left:before {
+  content: "\e917";
+  }
+  .icon-pointer-right:before {
+  content: "\e918";
+  }
+  .icon-csv:before {
+  content: "\e919";
+  }
+  .icon-zip:before {
+  content: "\e91a";
+  }
+  .icon-attention-sym:before {
+  content: "\e91c";
+  }
+  .icon-question-sym:before {
+  content: "\e91d";
+  }
+  .icon-star-solid:before {
+  content: "\e91e";
+  }
+  .icon-pause:before {
+  content: "\e91b";
+  }
+  .icon-openbox:before {
+  content: "\e91f";
+  }
+  .icon-resume:before {
+  content: "\e920";
+  }
+
+
+body{
+    background-color: #f1f1f4;
+    font-size:1em;
+}
+
+header .row{
+  margin:0;
+  padding:0;
+  background-color: #03BA5B;
+  padding:10px 0 0 20px;;
+  height:50px;
+  width:100%;
+  position:fixed;
+  z-index: 10;
+  box-shadow: 0px 0px 5px #414141;
+  font-size:12px;
+}
+
+
+.row{
```

Comment:
be **veeerrryyy** careful overriding bootstrap classes! This will make those changes to the entire app. Instead, you could make whats called a modifier class which modifies the styles of an element when you add its class. So in this example, you could make a class called `.row-with-space` or something and add it to all the places you need the extra padding top :)